src/coreclr/pal/prebuilt/inc/corprof.h

```diff
@@ -55,231 +55,238 @@
 #define __ICorProfilerCallback_FWD_DEFINED__
 typedef interface ICorProfilerCallback ICorProfilerCallback;
 
-#endif  /* __ICorProfilerCallback_FWD_DEFINED__ */
+#endif 	/* __ICorProfilerCallback_FWD_DEFINED__ */
```

Comment:
Is there a spaces->tabs conversion that is happening in this file? I'm not sure if you opened the file in your editor which did some automatic whitespace conversion, or something changed in the code generator, or some past checkin accidentally changed the whitespace and this is the first time the file is being regenerated since then.

Not a big deal but it would be nice to confirm that the whitespace that gets checked in is what was produced by the code generator and not an incidental change applied by an auto-formatting editor. It just adds a lot of noise to the diff.

Reply:
I believe it is a spaces->tabs conversion when I copied over the file via `cp ./artifacts/obj/coreclr/windows.x64.Debug/inc/idls_out/corprof.h ./src/coreclr/pal/prebuilt/inc/`

I just looked and maybe theres something different between my machine generating the `corprof.h` and @davmason's machine generating the file in #80528?

![Screenshot 2024-06-21 202115](https://github.com/dotnet/runtime/assets/16830051/4040df76-cd3d-4c9f-a0df-d806a0815ad9)

Maybe a configuration setting difference that I might have missed?

Reply:
The build generates the file with tabs (ie what you have in the PR) on my machine as well. My guess is that it was accidental edit that converted tabs to spaces.

Reply:
I think I have a setting that converts tabs to spaces automatically, it's probably my fault the file got checked in with spaces

Reply:
No worries David, we'll convert it back to tabs in this PR and then it should be stable after that.